migrations/versions/9155f0d3d028_.py

```diff
@@ -0,0 +1,132 @@
+"""v3.6: set TLS v1.0 for existing ldap resolvers as legacy behavior while setting TLS v1.3 as default in v3.6
+
+Revision ID: 9155f0d3d028
+Revises: d5870fd2f2a4
+Create Date: 2021-03-23 14:25:48.425762
+privacyIDEA Version: 3.6
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '9155f0d3d028'
+down_revision = 'd5870fd2f2a4'
+
+from alembic import op
+from sqlalchemy import orm
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.schema import Sequence
+
+Base = declarative_base()
+
+
+class Resolver(Base):
+    __tablename__ = 'resolver'
+    __table_args__ = {'mysql_row_format': 'DYNAMIC'}
+    id = sa.Column(sa.Integer, Sequence("resolver_seq"), primary_key=True,
+                   nullable=False)
+    name = sa.Column(sa.Unicode(255), default=u"",
+                     unique=True, nullable=False)
+    rtype = sa.Column(sa.Unicode(255), default=u"",
+                      nullable=False)
+    config_list = orm.relationship('ResolverConfig',
+                                  lazy='select',
+                                  backref='resolver')
+    realm_list = orm.relationship('ResolverRealm',
+                                 lazy='select',
+                                 foreign_keys='ResolverRealm.resolver_id')
+
+
+class ResolverConfig(Base):
+    __tablename__ = 'resolverconfig'
+    id = sa.Column(sa.Integer, Sequence("resolverconf_seq"), primary_key=True)
+    resolver_id = sa.Column(sa.Integer,
+                            sa.ForeignKey('resolver.id'))
+    Key = sa.Column(sa.Unicode(255), nullable=False)
+    Value = sa.Column(sa.Unicode(2000), default=u'')
+    Type = sa.Column(sa.Unicode(2000), default=u'')
+    Description = sa.Column(sa.Unicode(2000), default=u'')
+    __table_args__ = (sa.UniqueConstraint('resolver_id',
+                                          'Key',
+                                          name='rcix_2'),
+                      {'mysql_row_format': 'DYNAMIC'})
+
+
+class Realm(Base):
```

Comment:
One last trick for migration scripts. In fact you do not need the **exact** copy of the ORM, only the parts, that are used.
I.e. in a migration script you could only map columns, that you actually need.

I this case you can remove the table definitions ``Realm`` and ``ResolverRealm``, since you are not accessing these tables in the logic of your code.

However, this is defined in the realationship in line 35 ``realm_list``. But you could also safely remove the ``realm_list`` definition from the ``Resolver``  class.

I would do this to shorten the code of the migration script.

Check it out. This comes in quite handy also for other external scripts. Also the linotp-migration script does it in similar ways, since not the comple ORM needs to be mapped.